lib/inquisition/rubocop/vulnerability.rb

```diff
@@ -0,0 +1,52 @@
+module Inquisition
+  module Rubocop
+    class Vulnerability
+      LEVELS = {
+        refactor: :low,
+        convention: :low,
+        warning: :medium,
+        error: :high,
+        fatal: :high
+      }.freeze
+
+      DETAILED_MATCH_TABLE = {
+        'Bundler/DuplicatedGem' => :duplication,
+        'Bundler/InsecureProtocolSource' => :security,
+        'Metrics/LineLength' => :style,
+        'Migrations/RemoveIndex' => :performance
+      }.freeze
+
+      MATCH_TABLE = {
+        'Security' => :security,
+        'Metrics' => :complexity,
+        'Performance' => :performance
+      }.freeze
+
+      def initialize(file, offense)
+        @file = file
+        @offense = offense
+      end
+
+      def to_h
+        {
+          severity: LEVELS[offense.severity.name],
+          category: find_category(offense.cop_name),
+          path: file,
+          message: offense.message,
+          line: offense.line
+        }
+      end
+
+      private
+
+      attr_reader :file, :offense
+
+      def find_category(cop_name)
```

Comment:
Propose to create a separate class which will be responsible for the categorization, what do you think?